package.json

```diff
@@ -39,6 +39,7 @@
     "fs-extra": "^10.0.0",
     "git-state": "^4.1.0",
     "ignore": "^5.1.8",
+    "micromatch": "^4.0.4",
```

Comment:
**chore(blocking)**: it needs to be listed in the "[Built With](https://github.com/scolladon/sfdx-git-delta#built-with)" section of the README.md

Reply:
Done.